README.md

```diff
@@ -0,0 +1,8 @@
+# Jenkins Pipelin Input Step Plugin
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-input-step.svg)](https://plugins.jenkins.io/pipeline-input-step)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-input-step.svg?color=blue)](https://plugins.jenkins.io/pipeline-input-step)
+
+Adds the Pipeline step `input` to wait for human input or approval. For more information please check the [Pipeline syntax documentation](https://jenkins.io/doc/book/pipeline/syntax/#input).
```

Comment:
I don't think we want to link to https://jenkins.io/doc/book/pipeline/syntax/#input here since that is Declarative-specific.

```suggestion
Adds the Pipeline `input` step to wait for human input or approval.
```